modules/ps_shoppingcart/modal.tpl

```diff
@@ -58,18 +58,18 @@
               <div class="mb-3">
                 <div class="blockcart-modal__total product-subtotal">
                   <span class="label">{l s='Subtotal:' d='Shop.Theme.Checkout'}</span>
-                  <span class="value">{$cart.subtotals.products.value}</span>
+                  <span class="subtotals value">{$cart.subtotals.products.value}</span>
```

Comment:
Is the `subtotals` class coming from the module? Otherwise let's use BEM

Reply:
like `blockcart-modal__subtotals` as already done upper, even if it's not perfect

Reply:
@NeOMakinG This is a change added from here, I noticed that this class is missing https://github.com/PrestaShop/PrestaShop/commit/c7bd50606ae12afdb4b2cbb9f32d3ffc41a67a80

Reply:
@NeOMakinG :)

Reply:
Alright